src/components/IntroSection/IntroSection.js

```diff
@@ -9,7 +9,7 @@ function IntroSection() {
     <div className="introSection">
       <img src={penIntro} className="penIntro" alt="" />
```

Comment:
why IntroSection.js file is here? don't make changes in files that are not related to this branch.

undo this file changes from this branch.